<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeteoMozaicWidget</class>
 <widget class="QWidget" name="MeteoMozaicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="WidgetStyleSheet" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>186</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777192</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
#groupBox{
      border-style: outset;
     border-width: 2px;
     border-radius: 10px;
       font: bold 14px;
     min-width: 10em;
     padding: 6px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="10" column="0">
          <widget class="QLabel" name="labeTmin_4">
           <property name="minimumSize">
            <size>
             <width>186</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="Line" name="line_4">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(211, 215, 207);
</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="Line" name="line_9">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(211, 215, 207);

</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_14">
           <property name="styleSheet">
            <string notr="true">background-color: url(none);
border-color: rgba(255, 255, 255, 0);
</string>
           </property>
           <property name="text">
            <string>Temperature Max</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="labTmax_4">
           <property name="minimumSize">
            <size>
             <width>186</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_13">
           <property name="styleSheet">
            <string notr="true">background-color: url(none);
border-color: rgba(255, 255, 255, 0);
</string>
           </property>
           <property name="text">
            <string>Temperature Min</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="LabelDyn" name="labePre_4">
           <property name="minimumSize">
            <size>
             <width>186</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#labePre_4{
 background-color: white;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: blue;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>Prévisions Météo : Nuageux - orageux - ciel voilé - Ensoleillé  </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labDate_4">
           <property name="minimumSize">
            <size>
             <width>186</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="styleSheet">
            <string notr="true">background-color: url(none);
border-color: rgba(255, 255, 255, 0);
</string>
           </property>
           <property name="text">
            <string> Prevision</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Météo</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetStyleSheet</class>
   <extends>QWidget</extends>
   <header>widgetstylesheet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelDyn</class>
   <extends>QLabel</extends>
   <header>labeldyn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
